lib/spree_springboard/resource/export/order.rb

```diff
@@ -34,10 +34,19 @@ def after_sync(order)
         def springboard_invoice!(order)
           if springboard_can_invoice?(order)
             invoice_springboard_id = springboard_invoice_create!(order)
-
             springboard_invoice_line_items_create!(order, invoice_springboard_id) if invoice_springboard_id.present?
             springboard_invoice_complete!(order) if invoice_springboard_id.present?
             invoice_springboard_id
+          # If the order has been invoiced AND it's open:
+          # attempt to resync line_items and complete the order.
+          elsif springboard_invoiced(order)
+            invoice_springboard_id = order.child_springboard_id('invoice')
+            if invoice_springboard_id.present? && order.springboard_element[:status] != 'closed'
+              order.line_items.springboard_not_synced.
+              each { |line_item| line_item.springboard_export!(parent: order) }
```

Comment:
Can we indent this line just one level so it is more obvious that it is a continuation of the last one?

Reply:
Solved: https://github.com/JenniKayne/spree_springboard/pull/7/commits/3e28f20639ed7f1542aa1f930ddab2fe0db0460f